.github/workflows/build_vyper_list.yaml

```diff
@@ -0,0 +1,32 @@
+name: build-vyper-list
+
+on:
+  push:
+  schedule:
+    # every day at midnight
+    - cron: "0 0 * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Prerequisites
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y curl wget jq findutils coreutils sed
```

Comment:
I'm not sure whether `findutils` and `coreutils` are installed by default, so I add it anyway